pkg/node/manager.go

```diff
@@ -321,13 +336,18 @@ func updateTunnelMapping(n *Node, ip *net.IPNet) {
 // UpdateNode updates the new node in the nodes' map with the given identity.
 // When using DirectRoute RouteType the field ownAddr should contain the IPv6
 // address of the interface that can reach the other nodes.
-func UpdateNode(n *Node, routesTypes RouteType, ownAddr net.IP) {
+func UpdateNode(n *Node, routesTypes RouteType, ownAddr net.IP, source Source) {
 	clusterConf.Lock()
 	defer clusterConf.Unlock()
 
 	ni := n.Identity()
 
 	oldNode, oldNodeExists := clusterConf.nodes[ni]
+	// Ignore kubernetes updates if the node already exists
+	if oldNodeExists && source == FromKubernetes {
```

Comment:
This should check if the source of the old node is not Kubernetes. We still want to support updates of k8s node if no kvstore entry has been received so far